applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_explicit_dynamic_solver.py

```diff
@@ -9,6 +9,8 @@
 # Import base class file
 import structural_mechanics_solver
 
+# Import common utilities
+import common_methods_solvers
```

Comment:
```suggestion
from . import common_methods_solvers
```

this way it will also work with the final python-module-only python-interface